pages/common/glab-auth.md

```diff
@@ -0,0 +1,20 @@
+# glab auth
+
+> Authenticate with a GitLab host from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/auth/index.html#synopsis>.
+
+- Log in with interactive prompt:
+
+`glab auth login`
+
+- Log in with a token from standard input (created in https://gitlab.com/-/profile/personal_access_tokens):
```

Comment:
```suggestion
- Log in with a token (created in https://gitlab.com/-/profile/personal_access_tokens):
```

Reply:
In the case of `gh auth` @navarroaxel recommended instead to add the "from standard input" clause (see https://github.com/tldr-pages/tldr/pull/4839#discussion_r511519506)